src/extensions/default/SVGCodeHints/SVGAttributes.json

```diff
@@ -11,6 +11,15 @@
     "animate/fill": {
         "attribOptions": ["freeze", "remove"]
     },
+    "animateColor/fill": {
+        "attribOptions": ["freeze", "remove"]
+    },
```

Comment:
Earlier it was using alias in such situations, that was removed for readability. I should use `type` property for each of these attributes.